plugin/pkg/shoot/validator/admission.go

```diff
@@ -217,7 +219,7 @@ func (v *ValidateShoot) Admit(ctx context.Context, a admission.Attributes, o adm
 			return nil
 		}
 
-		if !reflect.DeepEqual(shoot.Spec.SeedName, oldShoot.Spec.SeedName) {
+		if a.GetSubresource() != "binding" && !reflect.DeepEqual(shoot.Spec.SeedName, oldShoot.Spec.SeedName) {
```

Comment:
Why this `a.GetSubresource() != "binding"` check is needed here? We already checked above(line 184) if the subresource is not binding this validation plugin will not run.

Reply:
This is to prevent Shoot updates from changing the seedName.

Reply:
What I mean is, that this is already checked above and doesn't need to be checked again.

Reply:
Nope, the code will reach here if the resource is Shoot and there is no subresource.